### DISCUSSION TITLE
Import issue with python <= 3.7

### DISCUSSION THREAD

**Discussion by elliotschrider** [optuna/optuna-dashboard#251] [General]
When using python with version <= 3.7 there is an error due to this import on line 5:
```from typing import Literal```
Changing this to ```from typing_extensions import Literal``` for python <= 3.7 allows me to use optuna dashboard perfectly.

An easy fix that works for all python versions is:
```
try:
    from typing import Literal
except ImportError:
    from typing_extensions import Literal
```
Would it possible to implement this fix? Thanks.

**Comment by c-bata** [comment]
Thank you for reporting the bug. Yes, you are absolutely correct. Could you open a pull request if you have time?

**Reply by c-bata** [reply]
I've just opened https://github.com/optuna/optuna-dashboard/pull/254 👍

**Reply by elliotschrider** [reply]
Forgot to write a thank you for this! Sorry that I was unable to submit the pull request before you, and thanks for taking the time to do it!

**Reply by c-bata** [reply]
No problem! Thank you for using optuna-dashboard 👍
